editor/src/clj/editor/properties_view.clj

```diff
@@ -240,7 +240,7 @@
                 update-fn (fn update-fn [_]
                             (let [property (property-fn)
                                   current-vals (properties/values property)
-                                  old-num (ffirst current-vals)
+                                  old-num (get (get current-vals 0) index)
```

Comment:
The main fix.
Since we check if old-value not equal to the new one, we have to get right `old-value`